### ISSUE TITLE
Problem with pyproj (from LatLon==1.0.1->-r requirements.txt (line 19))

### ISSUE THREAD

**Issue by felipefrancisco** [AHAAAAAAA/PokemonGo-Map#2900]
## Expected Behavior
EasySetup should complete without errors.

## Current Behavior
While executing it, this happens:

```
Collecting pyproj (from LatLon==1.0.1->-r requirements.txt (line 19))
  Using cached pyproj-1.9.5.1.tar.gz
    Complete output from command python setup.py egg_info:
    nad2bin.c
    nad2bin.c(129) : warning C4244: '=' : conversion from 'double' to 'float', possible loss of data
    nad2bin.c(130) : warning C4244: '=' : conversion from 'double' to 'float', possible loss of data
    nad2bin.c(137) : warning C4244: '=' : conversion from 'double' to 'float', possible loss of data
    nad2bin.c(138) : warning C4244: '=' : conversion from 'double' to 'float', possible loss of data
    nad2bin.c(352) : warning C4244: '=' : conversion from 'double' to 'float', possible loss of data
    nad2bin.c(353) : warning C4244: '=' : conversion from 'double' to 'float', possible loss of data
    pj_malloc.c
    using bundled proj4..
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "c:\users\felipe~1\appdata\local\temp\pip-build-cwl8yi\pyproj\setup.py", line 74, in <module>
        cc.link_executable(objects, execname, extra_postargs = [ '/MANIFEST' ] if os.name == 'nt' else  None)
      File "c:\python27\lib\distutils\ccompiler.py", line 700, in link_executable
        debug, extra_preargs, extra_postargs, None, target_lang)
      File "c:\python27\lib\distutils\msvc9compiler.py", line 669, in link
        raise LinkError(msg)
    distutils.errors.LinkError: command 'C:\\Users\\Felipe Francisco\\AppData\\Local\\Programs\\Common\\Microsoft\\Visual C++ for Python\\9.0\\WinSDK\\Bin\\mt.exe' failed with exit status -1073741819
```

## Possible Solution
?

## Steps to Reproduce (for bugs)
Just run easy setup or try to install using `pip install -r requirements.txt`

## Context
Just tried to install using the EasySetup, I was following the tutorial.

## Your Environment
* Version used: develop
* Environment name and version (e.g. Python 2.7): Python 2.7
* Operating System and version (desktop or mobile): Windows 10

**Comment by DhananiK** [comment]
How did you fix this?

**Comment by felipefrancisco** [comment]
Solved by installing Python 2.7.11
